sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -107,8 +107,6 @@ protected MetricsAdvisorAdministrationClient()
         /// <exception cref="ArgumentException"><paramref name="dataFeedId"/> is empty or not a valid GUID.</exception>
         public virtual async Task<Response<DataFeed>> GetDataFeedAsync(string dataFeedId, CancellationToken cancellationToken = default)
         {
-            Argument.AssertNotNullOrEmpty(dataFeedId, nameof(dataFeedId));
-
             Guid dataFeedGuid = ClientCommon.ValidateGuid(dataFeedId, nameof(dataFeedId));
```

Comment:
Most of this PR consists in removing these null-empty checks for our GUIDs. We don't need them because our helper method `ValidateGuid` already does this validation. Implemention [here](https://github.com/Azure/azure-sdk-for-net/blob/2387e787586af4e250740ac756984cfb9717cb97/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/ClientCommon.cs#L24).